src/test/java/com/google/jenkins/plugins/computeengine/integration/Commands.java

```diff
@@ -0,0 +1,22 @@
+package com.google.jenkins.plugins.computeengine.integration;
+
+import lombok.Getter;
+
+/**
+ * An enum defining commands that can be run on both linux and windows agents, with corresponding
+ * String format values for each platform.
+ */
+@Getter
+public enum Commands {
+  ECHO("echo %s", "echo %s"),
+  EXIT("exit /b %s", "exit %s"),
+  SLEEP("ping /n %s /l 1024 google.com ", "sleep %s");
```

Comment:
Yes, this is a little weird. I was not able to get the proper windows equivalent "timeout" to work. Looking around I found that there are some weird issues with running the command in non-interactive scripts, discussed in places such as this [comment](https://serverfault.com/questions/432322/how-to-sleep-in-a-batch-file#comment912165_432323).

Reply:
Makes sense. Is using Powershell a viable alternative to batch? If so, we could consider using -NonInteractive: https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Core/About/about_PowerShell_exe?view=powershell-5.1#-noninteractive

Reply:
Thanks for the tip, while I had trouble using powershell within the existing batch scripts, this led me to finding the Powershell plugin which automatically applies the NonInteractive flag when running scripts, and I was able to remove this hack.